superset/reports/notifications/slack.py

```diff
@@ -44,6 +44,7 @@
     NotificationParamException,
     NotificationUnprocessableException,
 )
+from superset.utils.core import get_email_address_list
```

Comment:
Likely not the best named function, but resolving that is best suited for another day.